Dockerfile

```diff
@@ -6,13 +6,16 @@ ENV PYTHONUNBUFFERED=1
 # Install Django
 RUN pip install \
     coverage \
-    django==2.2 \
+    django \
```

Comment:
Please pin the version , for example `django==3.0`
While you are at it, can you pin the versions for all the other packages?
That way we won't have any surprise with packages updated to new versions when we don't want them to be.